src/core/Akka.Persistence/AtLeastOnceDeliverySemantic.cs

```diff
@@ -380,8 +432,8 @@ public AtLeastOnceDeliverySemantic(IActorContext context, PersistenceSettings.At
         private void StartRedeliverTask()
         {
             if (_redeliverScheduleCancelable != null) return;
-            var interval = new TimeSpan(RedeliverInterval.Ticks / 2);
-            _redeliverScheduleCancelable = _context.System.Scheduler.ScheduleTellRepeatedlyCancelable(interval, interval, _context.Self,
+            var delay = new TimeSpan(RedeliverInterval.Ticks / 2);
+            _redeliverScheduleCancelable = _context.System.Scheduler.ScheduleTellRepeatedlyCancelable(delay, RedeliverInterval, _context.Self,
```

Comment:
Was this a bug? The first parameter is the initial delay of the very first callback. The second parameter is then the interval this will fire in, so this should be the whole `RedeliverInterval`, not half?

Reply:
yeah, must have been a bug. Nice catch.